### PR TITLE
Preping for 1.3.0 release

### PR DESCRIPTION
**Pull request by bdemers**
Take a look at the first commit: https://github.com/okta/okta-sdk-java/pull/227/commits/6e8cdd1f47e6c52533c58d4e31924f2cd126de65 the 2nd is automated

### CODE REVIEW

File: src/ci/common.sh [hunk 1]
```diff
@@ -18,14 +18,15 @@
 
 # deploy snapshot from ONLY this branch
 SNAPSHOT_BRANCH="master"
+PROJECT_NAME="okta-sdk-java"
```
[the remark this answers is not in the document]
Comment (bdemers) [reply src/ci/common.sh]: I don't think codacy has ever commented on a specific line before, that is pretty cool

**Comment by bdemers** [comment]
@jmelberg-okta note: the `codecov/project` check is a false positive do to minor differences in the IT coverage based on the retry logic.

File: src/ci/common.sh [hunk 2]
```diff
@@ -15,17 +15,20 @@
 # limitations under the License.
 #
 
+# ignore unused var in this script as this script defines common bits for our other scripts
+# shellcheck disable=SC2034
 
 # deploy snapshot from ONLY this branch
 SNAPSHOT_BRANCH="master"
+PROJECT_NAME="okta-sdk-java"
```
[the remark this answers is not in the document]
Comment (bdemers) [reply src/ci/common.sh]: this is a codacy config error, line 19 configures this file to ignore that pattern
I'm not going to waist any more time on this.  I think codacy is pretty cool it also isn't easy to configure...

File: src/ci/common.sh [hunk 3]
```diff
@@ -15,17 +15,20 @@
...
 
 # Get the slug from the TRAVIS var, or parse the 'origin' remote
 REPO_SLUG=${REPO_SLUG:-${TRAVIS_REPO_SLUG:-$(git remote get-url origin | sed 's_.*\:__; s_.*github.com/__; s_\.git__')}}
 PULL_REQUEST=${PULL_REQUEST:-${TRAVIS_PULL_REQUEST:-true}} # default to true
 BRANCH=${TRAVIS_BRANCH:-"$(git rev-parse --abbrev-ref HEAD)"}
 
 # run the ITs if we have an ENV_VARS are set
-if [ $TRAVIS_SECURE_ENV_VARS = true ] ; then
+if [ "$TRAVIS_SECURE_ENV_VARS" = true ] ; then
```
Comment (jmelberg-okta) [inline src/ci/common.sh]: nit: Use variable expansion here as you've done in other places

`"$TRAVIS_SECURE_ENV_VARS"` -> `"${TRAVIS_SECURE_ENV_VARS}"`

**Pull request merged**